projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -31,6 +31,7 @@
             [pseudoInvalid]="pseudoInvalid"
             [pseudoFocused]="pseudoFocused"
             [pseudoHovered]="pseudoHovered"
+            (paste.capture.prevent.stop)="onPaste($event.clipboardData.getData('text'))"
```

Comment:
i am not sure that it is good enough 🤔

Maybe lets move all extraction of the event data inside onPaste method ?

Reply:
thanks, done